onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -111,55 +113,24 @@ def create_backend_test(testname=None):
-                                 '^test_range_float_type_positive_delta_cpu',
-                                 '^test_range_float_type_positive_delta_expanded_cpu',
-                                 '^test_range_int32_type_negative_delta_cpu',
-                                 '^test_range_int32_type_negative_delta_expanded_cpu',
-                                 '^test_det_2d_cpu',
-                                 '^test_det_nd_cpu',
-                                 '^test_resize_downsample_scales_cubic_A_n0p5_exclude_outside_cpu',
-                                 '^test_resize_downsample_scales_cubic_align_corners_cpu',
-                                 '^test_resize_downsample_scales_cubic_cpu',
-                                 '^test_resize_downsample_scales_linear_align_corners_cpu',
-                                 '^test_resize_downsample_scales_linear_cpu',
-                                 '^test_resize_downsample_scales_nearest_cpu',
-                                 '^test_resize_downsample_sizes_cubic_cpu',
-                                 '^test_resize_downsample_sizes_linear_pytorch_half_pixel_cpu',
-                                 '^test_resize_downsample_sizes_nearest_cpu',
-                                 '^test_resize_downsample_sizes_nearest_tf_half_pixel_for_nn_cpu',
-                                 '^test_resize_tf_crop_and_resize_cpu',
-                                 '^test_resize_upsample_scales_cubic_A_n0p5_exclude_outside_cpu',
-                                 '^test_resize_upsample_scales_cubic_align_corners_cpu',
-                                 '^test_resize_upsample_scales_cubic_asymmetric_cpu',
-                                 '^test_resize_upsample_scales_cubic_cpu',
-                                 '^test_resize_upsample_scales_linear_align_corners_cpu',
-                                 '^test_resize_upsample_scales_linear_cpu',
-                                 '^test_resize_upsample_scales_nearest_cpu',
-                                 '^test_resize_upsample_sizes_cubic_cpu',
-                                 '^test_resize_upsample_sizes_nearest_ceil_half_pixel_cpu',
-                                 '^test_resize_upsample_sizes_nearest_cpu',
-                                 '^test_resize_upsample_sizes_nearest_floor_align_corners_cpu',
-                                 '^test_resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric_cpu',
-                                 '^test_sequence_*',
-                                 '^test_scatter_.*',
-                                 '^test_edge_pad_cpu.*',  # test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests
-                                 '^test_reflect_pad_cpu.*'  # test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests
-        ]
+                                 '^test_resize_downsample_scales_cubic_A_n0p5_exclude_outside_cpu',  # not implemented yet
```

Comment:
Can you please remove '^test_resize_downsample_scales_cubic_A_n0p5_exclude_outside_cpu',  and '^test_resize_upsample_scales_cubic_A_n0p5_exclude_outside_cpu',  
from this list... I removed it from main.cc but missed this file

Reply:
Removed.